rust/ql/test/library-tests/dataflow/sources/test_futures_io.rs

```diff
@@ -1,18 +1,18 @@
-fn sink<T>(_: T) { }
+fn sink<T>(_: T) {}
 
 // --- tests ---
 
-use std::pin::Pin;
-use std::task::{Context, Poll};
-use std::io;
-use futures::io::AsyncRead;
-use futures::io::AsyncReadExt;
+use async_std::net::TcpStream;
+use async_std::sync::Arc;
 use futures::io::AsyncBufRead;
 use futures::io::AsyncBufReadExt;
+use futures::io::AsyncRead;
+use futures::io::AsyncReadExt;
 use futures::StreamExt;
-use futures_rustls::{TlsConnector};
-use async_std::sync::Arc;
-use async_std::net::TcpStream;
+use futures_rustls::TlsConnector;
+use std::io;
+use std::pin::Pin;
+use std::task::{Context, Poll};
```

Comment:
Why are we changing the imports for this test?

If it was done by an auto-formatter can be avoid doing that in future, we do not want to test only auto-formatted target code.

Reply:
Yes, it was auto-format.